src/org/zaproxy/zap/extension/pscan/PassiveScanThread.java

```diff
@@ -141,7 +146,13 @@ public void run() {
 								if (shutDown) {
 									return;
 								}
-								if (scanner.isEnabled() && scanner.appliesToHistoryType(href.getHistoryType())) {
+								int hrefHistoryType = href.getHistoryType();
+								if (pscanOptions.isScanFuzzerMessages()) {
+									addApplicableHistoryType(HistoryReference.TYPE_FUZZER);
+									addApplicableHistoryType(HistoryReference.TYPE_FUZZER_TEMPORARY);
+								}
+								if (scanner.isEnabled() && scanner.appliesToHistoryType(hrefHistoryType)
```

Comment:
It needs parentheses for the types (otherwise the enabled could be bypassed, or was that intended?).